src/App.js

```diff
@@ -1,23 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
+import ArticleList from './components/ArticleList';
+import { useState } from 'react';
+import { Typography, CssBaseline } from '@material-ui/core'
 
 function App() {
+  const [articles, setArticles] = useState([]);
```

Comment:
does `articles` state belong in `<App>`?

Make sure you are using state as low down in your component tree as possible